backend/src/api/routes/unified_guide.py

```
@@ -0,0 +1,154 @@
+    try:
+        pdf_service = MagazinePDFService(destination=request.destination)
+        
+        pdf_bytes = pdf_service.create_magazine_pdf(request.guide_data)
+        
+        return {
+            "success": True,
+            "pdf_size": len(pdf_bytes),
+            "message": "Magazine-quality PDF generated successfully"
+        }
+        
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)